app/src/globals/functions.ts

```diff
@@ -0,0 +1,86 @@
+import { faFileImage, faFilePdf, faFileWord, faFilePowerpoint, faFileExcel, faFileCsv, faFileAudio, faFileVideo, faFileArchive, faFileCode, faFileAlt, faFile} from '@fortawesome/free-solid-svg-icons'
+import { CancelToken } from "axios";
+import Api from "../controllers/Api";
+
+
+/**
+*  Get the URL for the resource in AdamRMS s3 Bucket
+ * @param fileid s3files_id
+ * @param size s3files_meta_size 
+ * @returns string 
+ */
+export async function s3url (fileid: string, size: number, cancelToken: CancelToken) {
```

Comment:
Could we bundle up some of these globals into a utility class?

I'm thinking s3url and the font awesome thing could go into a files class for example

Reply:
If you end up with a class of all static functions in the class you may as well just have the functions cause then surely you can import just the ones you need rather than a very large class?

Reply:
True, can I revise to splitting it up into separate files

Reply:
so are we splitting these “global” functions into separate classes?